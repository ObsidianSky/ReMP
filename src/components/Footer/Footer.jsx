import React from 'react';
if (process.env.BROWSER) {
	require('./Footer.scss');
}
import Logo from '../common/Logo/Logo';

const Footer = () => (
    <div className="footer">
        <Logo/>
    </div>
);

export default Footer;