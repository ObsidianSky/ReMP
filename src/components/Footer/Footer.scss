@import '../../assets/styles/variables';

.footer {
  background: $footer-bg;
  padding: 20px;

  &__inner {
    max-width: 1200px;
    margin: 0 auto;
  }
}