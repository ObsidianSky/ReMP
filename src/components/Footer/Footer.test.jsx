import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Footer from './Footer';

describe('Footer', () => {

    test('should renders correctly', () => {
        const wrapper = shallow(<Footer />);

        expect(toJson(wrapper)).toMatchSnapshot();
    });
});