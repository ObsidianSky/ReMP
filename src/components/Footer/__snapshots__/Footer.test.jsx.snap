// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should renders correctly 1`] = `
<div
  className="footer"
>
  <Logo />
</div>
`;
